draft-ietf-tsvwg-careful-resume.xml

```diff
@@ -1011,13 +1011,43 @@ Examples of the transitions between phases are provided in <xref target="Example
 		<figure anchor="qlog-def">
 	         <!--- 	<name>cr_phase</name> -->
 <sourcecode type="cddl"><![CDATA[
-XX Fix me as CDDL XX
-<!-- CrState::OBSERVE => { 0 }
-CrState::RECON => { 1 }
-CrState::UNVAL => { 2 }
-CrState::VALIDATE => { 3 }
-CrState::NORMAL => { 4 }
-CrState::RETREAT => { 100 } -->
+RecoveryCarefulResumePhaseUpdated = {
+? old: CarefulResumePhase,
+new: CarefulResumePhase,
+
+
+state_data: CarefulResumeStateParameters,
+? restored_data: CarefulResumeRestoredParameters,
+
+
+? trigger: "packet_loss"	;Loss triggers moving to safe retreat
+"cwnd_limited"			;trigger for moving to unvalidated
+"cr_mark_acknowledged"		;trigger for moving to validating or normal
+"rtt_not_validated"		;trigger for moving to normal, when CR not allowed
+"ECN_CE"			;trigger for moving to safe retreat.
+"exit_recovery"			;trigger for moving to normal, 1 rtt after a congestion event 
```

Comment:
The "house style" that we landed on was to use newlines more egregiously - splitting comments on to their own lines, one type or variant per line.  Newlines also help avoid the dreaded 72 column width police.

House styl also requires a space after `;`, and more use of indentation on types .

More importantly, this is not actually valid CDDL because you need a trailing `/` after each value. 

See some examples in https://github.com/quicwg/qlog/blob/main/draft-ietf-quic-qlog-quic-events.md?plain=1#L398 might help you. To get you started, something like below, but the PR needs a full sweep

```
? trigger:
        ; Loss triggers moving to safe retreat
        "packet_loss"
        ; trigger for moving to unvalidated
        "cwnd_limited" /
        ...
... 
```

Also, I'd remove the redundant `trigger` in each comment. The type is a trigger so the comment is redundant and adds more text.